### PR TITLE
Updated biblib API docs and examples with raw parameter.

### PR DESCRIPTION
**Pull request by tjacovich**

### CODE REVIEW

File: API_documentation_Python/Libraries_API_Python.ipynb [hunk 1]
```diff
@@ -261,6 +265,155 @@
    ]
   },
   {
+   "attachments": {},
+   "cell_type": "markdown",
```
Comment (kelockhart) [inline API_documentation_Python/Libraries_API_Python.ipynb:269]: This section is now duplicated within the notebook - I think a copy/paste went awry.

File: API_documentation_Python/Libraries_API_Python.ipynb [hunk 2]
```diff
@@ -261,6 +265,155 @@
...
+    "                       headers={'Authorization': 'Bearer ' + token})\n",
+    "results.json()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "using the `raw` query parameter allows the user to return the exact bibcodes present in the library, regardless of whether or not they are present in ADS. "
```
Comment (kelockhart) [inline API_documentation_Python/Libraries_API_Python.ipynb:358]: Capitalize this - it sort of blends into the request response above when rendered.

File: API_documentation_Python/Libraries_API_Python.ipynb [hunk 3]
```diff
@@ -261,6 +265,155 @@
...
+   ]
```
Comment (kelockhart) [inline API_documentation_Python/Libraries_API_Python.ipynb:359]: Can you also add this to the Unix notebook?

**Review by kelockhart — CHANGES_REQUESTED** [review]
Some copy/paste issues, plus can you add the example to the Unix notebook as well?

File: API_documentation_Python/Libraries_API_Python.ipynb (hunk 1 above)
Comment (tjacovich) [reply API_documentation_Python/Libraries_API_Python.ipynb]: Sorry about that. Completely missed the duplicate section.

File: API_documentation_Python/Libraries_API_Python.ipynb (hunk 2 above)
Comment (tjacovich) [reply API_documentation_Python/Libraries_API_Python.ipynb]: Capitalized and added to the unix notebook

**Comment by tjacovich** [comment]
Sorry about the duplication. Completely overlooked that. The updated text and example are now in the Unix notebook as well.

**Review by kelockhart — APPROVED** [review]

**Pull request merged**